website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -41,12 +41,16 @@
         <br />
         <br />
         <div class="row" data-bind="if: showSelect">
-          <div class="col-md-8">
+          <div class="col-md-6">
```

Comment:
Same here. I'm not sure this markup is correct; do you need a `form-group` here?